packages/presentation/lib/screen/details/details_screen.dart

```diff
@@ -71,33 +72,37 @@ class _DetailsScreenState extends BlocScreenState<DetailsScreen, DetailsBloc> {
                       ),
                     ]),
                 body: SingleChildScrollView(
-                  child: Stack(children: [
-                    ImageFiltered(
-                      imageFilter: ImageFilter.blur(sigmaY: 3, sigmaX: 3),
-                      child: Container(
-                        height: Dimens.size260,
-                        decoration: BoxDecoration(
-                          image: DecorationImage(
-                            image: NetworkImage('${screenData?.movie?.image}'),
-                            fit: BoxFit.fitWidth,
-                            alignment: FractionalOffset.topCenter,
+                  child: Column(
+                    children: [
+                      Stack(
+                        children: [
+                          ImageFiltered(
+                            imageFilter: ImageFilter.blur(
+                              sigmaY: 3,
+                              sigmaX: 3,
+                            ),
+                            child: Container(
+                              height:
+                                  Responsive.backgroundDetailsImage(context),
```

Comment:
need to think about how can we refactor it. I don't think that's a good idea to put everything into a Responsive class. seems that this class should contain only helpful utils to work with desktop